es6.js

```diff
@@ -7,35 +7,33 @@ module.exports = {
   plugins: ['@babel', 'html'],
   extends: ['airbnb-base', 'plugin:import/typescript'],
   env: {
-    es6: true,
+    es2021: true,
   },
   parser: '@babel/eslint-parser',
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: 'latest',
     sourceType: 'module',
     requireConfigFile: false,
   },
-  settings: {
-    'import/extensions': ['.js', '.json', '.ts', '.tsx'],
-  },
   rules: {
     'no-console': ['warn', { allow: ['error', 'trace', 'time'] }],
     'no-param-reassign': 'off',
     'no-plusplus': 'off',
-    'no-unused-expressions': 'off', // Disable for better @babel version
-    '@babel/no-unused-expressions': ['warn', { allowShortCircuit: true }],
     'no-use-before-define': ['error', { functions: false, classes: true, variables: true }],
-    'no-warning-comments': 'warn',
+    'no-warning-comments': ['warn', { terms: ['FIXME', 'TODO', 'HACK'], location: 'anywhere' }],
```

Comment:
Note from Jordi:
no-warning-comments -- looks for todo, fixme, or hack anywhere in the comment now rather than the default of todo, fixme, xxx at the start of a comment.